desktop-app/app/components/DeviceManager/DeviceList/index.js

```diff
@@ -10,91 +10,109 @@ import DeviceItem from '../DeviceItem';
 
 import styles from './styles.css';
 
-export default function DeviceList({
-  droppableId,
-  devices,
-  enableFiltering,
-  onFiltering,
-  enableCustomDeviceDeletion,
-  deleteDevice,
-}) {
-  const [searchOpen, setSearchOpen] = useState(false);
-  const [searchText, setSearchText] = useState('');
-  const [filteredDevices, setFilteredList] = useState(devices);
+const DeviceListHandler = props => {
```

Comment:
Why all these changes related to the filtered devices?

Reply:
Fair question. It's because the DeviceManager (where we want to get a keypress) and the DeviceList (containing a search box) are separate components. This means that in order to focus the search box within the DeviceManager we have to have a reference to the DeviceList. The problem is you can't use a ref attribute on a functional component meaning I had to convert the DeviceList into a class component. Because I had trouble getting device filtering working using the class component, I split the device filtering into its own functional handler, meaning most of the code within it, is basically what was there before.

Reply:
@jzabala @manojVivek  should we go with this?
@maxwillkelly please solve the conflicts by merging `master` into this branch

Reply:
No problem will get that done soon